### PR TITLE
add a bulk delete example

### PR DESCRIPTION
**Pull request by igorbernstein2**

### CODE REVIEW

File: java/dataflow-bulk-delete/src/main/java/com/google/cloud/bigtable/example/bulk_delete/simple/steps/ScanPrefixDoFn.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  @ProcessElement
+  public void processElement(ProcessContext c) throws IOException {
+    String prefix = c.element();
+    byte[] prefixBytes = prefix.getBytes();
+
+    byte[] start = prefixBytes;
+    byte[] end = RowKeyUtil.calculateTheClosestNextRowKeyForPrefix(start);
+
+    Scan scan = new Scan(start, end)
```
Comment (sduskis) [inline java/dataflow-bulk-delete/src/main/java/com/google/cloud/bigtable/example/bulk_delete/simple/steps/ScanPrefixDoFn.java]: scan.setRowPrefixFilter(prefixBytes)

File: java/dataflow-bulk-delete/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+This demonstrates how to delete a lot of data to workaround the rate limiting of DropRowRanges. 
+There are 2 examples provided simple and advanced. Both accomplish the same thing.
+
+## Simple
+
+This is a bare bones example to show the basic idea of taking a list of row prefixes and
+scanning Bigtable for matching rows and deleting those row in bulk.
+
+
+## Advanced
```
Comment (sduskis) [inline java/dataflow-bulk-delete/README.md]: Can we please get rid of Advanced?  It's too complex for what we're trying to achieve.

Comment (igorbernstein2) [reply java/dataflow-bulk-delete/README.md]: Done

File: java/dataflow-bulk-delete/src/main/java/com/google/cloud/bigtable/example/bulk_delete/simple/steps/ScanPrefixDoFn.java (hunk 1 above)
Comment (igorbernstein2) [reply java/dataflow-bulk-delete/src/main/java/com/google/cloud/bigtable/example/bulk_delete/simple/steps/ScanPrefixDoFn.java]: Done

**Comment by sduskis** [comment]
@igorbernstein2 ping.

**Pull request merged**